app/views/pages/enrolment/non_gcse.html.erb

```diff
@@ -12,7 +12,7 @@
             and support you on your journey.
           </p>
 
-          <a href="https://forms.office.com/e/VHwFA9aS3g" class="govuk-button button">Request SME support</a>
+          <a href="https://forms.office.com/e/VSLpEXu3vH" class="govuk-button button">Request SME support</a>
```

Comment:
I've seen this link a few times around the app - could we move it into the external_link_helper?

Reply:
Good point, will do that now